tools/gmhazard_scripts/ensemble_creation/gmm_weights_21p10.yaml

```diff
@@ -1,15 +1,35 @@
+CAV:
+        !!python/tuple [ACTIVE_SHALLOW, SUBDUCTION_INTERFACE, SUBDUCTION_SLAB, VOLCANIC]:
+                CB_10: 1
+AI:
+        !!python/tuple [ACTIVE_SHALLOW, SUBDUCTION_INTERFACE, SUBDUCTION_SLAB, VOLCANIC]:
+                CB_12: 1
+!!python/tuple [Ds575, Ds595]:
+        !!python/tuple [ACTIVE_SHALLOW, SUBDUCTION_INTERFACE, SUBDUCTION_SLAB, VOLCANIC]:
+                AS_16: 1
 !!python/tuple [pSA, PGA]:
         ACTIVE_SHALLOW:
                 Br_10: 0.2
                 ASK_14: 0.2
                 BSSA_14: 0.2
                 CB_14: 0.2
                 CY_14: 0.2
-        SUBDUCTION:
+        SUBDUCTION_INTERFACE:
+                ZA_06: 0.1
+                BCH_16: 0.05
+                A_18: 0.1
+                P_20: 0.3
+                AG_20: 0.075
+                AG_20_NZ: 0.075
+                K_20: 0.1
+                K_20_NZ: 0.2
+        SUBDUCTION_SLAB:
                 ZA_06: 0.1
+                BCH_16: 0.05
+                A_18: 0.1
                 P_20: 0.2
-                AG_20: 0.175
-                AG_20_NZ: 0.175
+                AG_20: 0.1
+                AG_20_NZ: 0.1
```

Comment:
This section shouldn't be updated. The prior weights were correct for "SUBDUCTION".